input/postgres/server_stats.go

```diff
@@ -111,10 +111,6 @@ func GetServerStats(ctx context.Context, c *Collection, db *sql.DB, ps state.Per
 	if err != nil {
 		return ps, ts, err
 	}
-	ps.ServerIoStats, err = GetPgStatIo(ctx, c, db)
```

Comment:
Memo: I was wondering what this is - it's introduced back in March in https://github.com/pganalyze/collector/pull/553

This is one of the stats that we collect once a min too (hence makes sense to move it to input/full_1min.go), and looks like we are not using in the app side yet.